k8s/helm/metaflow/charts/metaflow-ui/templates/backend_deployment.yaml

```diff
@@ -60,20 +58,22 @@ spec:
             - name: PATH_PREFIX
               value: "/api"
             - name: MF_DATASTORE_ROOT
-              value: {{ .Values.METAFLOW_DATASTORE_SYSROOT_S3 | quote }}
+              value: {{ .Values.uiBackend.metaflowDatastoreSysRootS3 | quote }}
             - name: METAFLOW_DATASTORE_SYSROOT_S3
-              value: {{ .Values.METAFLOW_DATASTORE_SYSROOT_S3 | quote }}
+              value: {{ .Values.uiBackend.metaflowDatastoreSysRootS3 | quote }}
+            - name: METAFLOW_S3_ENDPOINT_URL
+              value: {{ .Values.uiBackend.metaflowS3EndpointURL | quote }}
```

Comment:
Should we set this as a conditional? Coz otherwise many deployments may actually not end up having this.

Reply:
good catch

Reply:
added